src/ToeplitzMatrices.jl

```diff
@@ -102,6 +102,10 @@ function mul!(y::StridedVector{T}, A::AbstractToeplitz{T}, x::StridedVector, α:
         return y
     end
 end
+# Avoid ambiguity error
+mul!(y::StridedVector{T}, A::AbstractToeplitz{T}, x::StridedVector, α::T, β::T) where {T<:Number} =
```

Comment:
Could `T<:Number` just be added to the above definition instead?

Reply:
I'm not completely sure that the elements are always `Number`s (although it's probably unlikely) which was why I added a separate method.

Reply:
Is there a reason they are specialised to `StridedVector`? The code looks like it should work regardless. (If you are adding the unnecessary restriction to `StridedVector`, might as well go all the way and do `StridedVector{<:Number}`).

Reply:
It's been a long time so I don't remember the details but when I wrote these, I only had floats in mind so if you don't rely on non-Numbers here then we can just restrict it and make a new minor release.